pkg/crc/constants/constants_windows.go

```diff
@@ -5,4 +5,6 @@ const (
 	PodmanRemoteExecutableName = "podman.exe"
 	TapSocketPath              = ""
 	DaemonHTTPNamedPipe        = `\\.\pipe\crc-http`
+	DaemonTaskName             = "crcDaemon"
+	AdminHelperServiceName     = "CodeReadyContainersAdminHelper"
```

Comment:
can we rename this to just `crcAdminHelper` ?

Reply:
It would be changed in the installer also, this service name is part of installer https://github.com/code-ready/crc/blob/main/packaging/windows/product.wxs.template#L43 , Also if we change the name we have to make sure older service shouldn't exist.

Reply:
@gbraad Can we have an follow up issue for name change and not block this PR for this one?

Reply:
@gbraad #3092